.travis.yml

```diff
@@ -39,6 +39,10 @@ jobs:
           env: DMD=2.078.3.s* F=production
         - <<: *test-matrix
           env: DMD=2.078.3.s* F=devel
+        - <<: *test-matrix
+          env: DMD=2.085.* F=production
```

Comment:
Is there a reason for using 2.085 in particular and not 2.086?

Reply:
There wasn't a specific reason. I've changed it to 2.086.*.